Sources/CovidCertificateSDK/TrustList/RevocationListUpdate.swift

```diff
@@ -70,23 +75,12 @@ class RevocationListUpdate: TrustListUpdate {
                 return .NETWORK_PARSE_ERROR
             }
 
-            let semaphore = DispatchSemaphore(value: 0)
-            var outcome: Result<RevocationList, JWSError> = .failure(.SIGNATURE_INVALID)
-
-            TrustlistManager.jwsVerifier.verifyAndDecode(httpBody: d) { (result: Result<RevocationList, JWSError>) in
-                outcome = result
-                semaphore.signal()
-            }
-
-            semaphore.wait()
-
-            guard let result = try? outcome.get() else {
+            guard let result = decoder.decode(d) else {
                 return .NETWORK_PARSE_ERROR
             }
 
-            _ = trustStorage.updateRevocationList(result)
-
-            nextSince = nextSinceHeader
+            let success = trustStorage.updateRevocationList(result, nextSince: nextSinceHeader)
```

Comment:
If updating the revocation list fails (for example while saving) do we really want to continue (with the loop)?

Reply:
The `nextSince` header is stored alongside the actual revocations. Therefore even if the saving fails we still want to fetch the complete revocation list and keep it in memory. On the next app start the list would get fetched again and we would try to save it.

Reply:
The alternative would be to return a new error case if we are unable to write to the disk.

Reply:
True, I did not remember the in-memory storage. In that case I don't see a reason to change it.